contributing.md

```diff
@@ -169,16 +181,12 @@ Generates a changefile based on the dev branch. This should be run on your PR be
 
 Checks that changefiles have been generated for relevant changed packages. Used by the CI tool on PRs, can also be used locally.
 
-- `npm run beachball:publish`
+- `npm run beachball:bump`
 
-Bumps versions and publishes all packages that have relevant changefiles. If the changefiles types are set to `None`, no version bump or publish is done. Run on push to the `master` branch.
+Bumps versions for packages with relevant changefiles. Run on `master` branch to check which packages are slated for release.
 
-- Notes
-   - Ensure you have the correct email set in your local git config.
```

Comment:
This is good info to keep around. I would leave it in personally

Reply:
It's listed in one of the sections above.